packages/teleport/src/services/mfa/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { Auth2faType } from 'shared/services';
+import { Option } from 'shared/components/Select';
+
+export type MfaOption = Option<Auth2faType>;
+
+export function getMfaOptions(auth2faType: Auth2faType) {
+  const mfaOptions: MfaOption[] = [];
+  const mfaEnabled = auth2faType === 'on' || auth2faType === 'optional';
+
+  if (auth2faType === 'optional' || auth2faType === 'off') {
```

Comment:
shouldn't `auth2faType === 'off'` return an empty array?

Reply:
When `auth2faType === 'off'` there won't be a dropdown shown, but behind the scenes this allows us to use `if(mfaOption.value === 'optional)'` both when the 'None' option is selected and when `auth2faType === 'off'` rather than needing an additional check for the latter